skyportal/plot.py

```diff
@@ -1336,6 +1336,9 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
 
     # These values are equivalent from the photometry plot values
```

Comment:
Not anymore?

Reply:
Oh woops! Wanna push your suggested change up to master?